README.md

```diff
@@ -1,4 +1,4 @@
-# Presto
+# Presto!
```

Comment:
Why add the exclamation point as the only change to the Presto README?

Reply:
No actually I wanted to retrigger the tests so made a dummy commit - will revert it.